ivy/functional/backends/paddle/general.py

```diff
@@ -81,10 +81,13 @@ def to_numpy(
         else:
             return x
     elif paddle.is_tensor(x):
+        dtype = ivy.as_ivy_dtype(x.dtype)
+        if dtype == "bfloat16":
```

Comment:
probably we can simplify this further unless I'm missing something, what do you think?
```
if dtype == "bfloat16":
    x = x.astype("floar32")
else:
    return np.asarray(x)
return np.asarray(x).astype("bfloat16")
```

Reply:
this ignores the `copy` arguments, I guess it's fine the way it's as it's  fairly understandable

Reply:
yep sounds good!